rules/docker-base.mk

```diff
@@ -2,6 +2,7 @@
 
 DOCKER_BASE = docker-base.gz
 $(DOCKER_BASE)_PATH = $(DOCKERS_PATH)/docker-base
+$(DOCKER_BASE)_DEPENDS += $(SUPERVISOR)
```

Comment:
it is used by every docker so I think it makes sense to be included in the docker-base.